apps/enterprise-knowledge-retrieval/README.md

```diff
@@ -0,0 +1,36 @@
+# Enterprise Knowledge Retrieval
+
+This repo is a deep dive on Enterprise Knowledge Retrieval, which aims to take some unstructured text documents and create a usable knowledge base application with it.
+
+This repo contains a notebook and a basic Streamlit app:
+- `enterprise_knowledge_retrieval.ipynb`: A notebook containing a step by step process of tokenising, chunking and embedding your data in a vector database, building a chat agent on top and running a basic evaluation of its performance
```

Comment:
second bullet ends in a period, first lacks a period

Reply:
Done